package.json

```diff
@@ -1,18 +1,28 @@
 {
   "name": "fastify-cli",
-  "version": "4.0.0",
+  "version": "3.1.0",
   "description": "Run a fastify route with one command!",
-  "main": "cli.js",
+  "main": "dist/index.js",
   "bin": {
-    "fastify": "cli.js"
+    "fastify": "./bin/run"
   },
+  "files": [
+    "/bin",
+    "/dist",
+    "/templates",
+    "/oclif.manifest.json"
+  ],
   "scripts": {
-    "lint": "standard",
-    "lint:fix": "standard --fix",
-    "unit": "tap \"test/**/*.test.{js,ts}\" \"templates/**/*.test.{js,ts}\"",
-    "pretest": "xcopy /e /k /i . \"..\\node_modules\\fastify-cli\" || rsync -r --exclude=node_modules ./ node_modules/fastify-cli || echo 'this is fine'",
-    "test": "npm run lint && npm run unit && npm run test:typescript",
-    "test:typescript": "tsd templates/plugin && tsc --project templates/app-ts/tsconfig.json && del-cli templates/app-ts/dist"
+    "build": "shx rm -rf dist && tsc -p tsconfig.json",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "npm run lint --fix",
+    "prepack": "npm run build",
+    "pack": "oclif manifest && oclif readme",
+    "postpack": "shx rm -f oclif.manifest.json",
+    "preunit": "npm run pack",
+    "unit": "tap \"test/**/*.test.ts\"",
```

Comment:
Any particular reason to go for tap here? E. g. mocha has way wider mainstream usage.

Reply:
`tap` is used through out the whole organization. So, it would be the first priority one.
`mocha` and `jest` can be added, but it needs somebody effort.

Reply:
There is a difference between internal and external use, no? CLI is not intended to be used primarily internally.

Reply:
Using united test framework inside the organization (`fastify`) have some advantages. Developer inside the organization do not need to learn multiple test framework.

> There is a difference between internal and external use, no?

I think it would be a long discussion on test framework topic.